README.md

```diff
@@ -130,9 +131,10 @@ For example, to access the user's API key is `ViewCustomize.context.user.apiKey`
 
 * [onozaty/redmine\-view\-customize\-scripts: Script list for "Redmine View Customize Plugin"](https://github.com/onozaty/redmine-view-customize-scripts)
 
-## Redmine 2.0.x or later
+## Supported versions
 
-* Redmine 2.0.x or later
+* Current version : Redmine 3.0.x or later
```

Comment:
Maybe init.rb should have `requires_redmine version_or_higher: '3.0.0'` .

* http://yardoc.redmine.jp/Redmine/Plugin.html#requires_redmine-instance_method
* http://redmine.jp/tech_note/plugins/developer/quickref/requires_redmine/

Reply:
I added at ebdb835.  Thanks.